### DISCUSSION TITLE
Let's test how this works

### DISCUSSION THREAD

**Discussion by kt97679** [ForthHub/discussion#91] [General]
Hello everybody, this is a test message.

**Comment by massung** [comment]
I got an email and didn't even "sign up" or participate in this thread beforehand. Seems good to me.

**Comment by larsbrinkhoff** [comment]
Sorting is supported, so that's good:  
https://github.com/ForthHub/discussion/search?q=participate&type=discussions

**Comment by mitra42** [comment]
Interesting that the numbers are contiguous with issues, so it looks like they are using the same mechanism (i.e. this discussion is #91 and most recent issue is #90)

**Comment by mitra42** [comment]
Also .... email notifications seem to be turned on by default, i.e. because I had email notifications for issues turned on, I got an email notification of the discussion posts.
